src/codemagic/apple/resources/enums.py

```diff
@@ -10,6 +10,18 @@ def __str__(self):
         return str(self.value)
 
 
+class BuildOrdering(_ResourceEnum):
```

Comment:
Basically needed to translate cli argument values e.g. PRE_RELEASE_VERSION to order api values e.g. preReleaseVersion

Reply:
no ordering, not relevant anymore:)